styles.css

```diff
@@ -0,0 +1,881 @@
+}
+
+.feature-logo {
+  display: none;
+  font-family: tnw, 'Times New Roman', Times, serif;
+}
+
+.main-feature,
+.sub-feature {
+  height: 150px;
+}
+
+.main-feature h3,
+.sub-feature h3 {
+  margin: 0;
+  padding: 0;
+}
+
+.menu-links a {
+  text-transform: uppercase;
+  font-size: 14px;
+  font-family: 'Courier New', Courier, monospace;
+  color: whitesmoke;
+  margin: 5px;
+  font-weight: 600;
+}
+
+.main-feature h3 a,
+.sub-feature h3 a {
+  color: #fff;
+}
+
+.sub-feature {
+  flex-direction: column;
+}
+
+.tnw-card {
+  position: relative;
+  height: 100%;
+  width: 100%;
+}
+
+.overlay-main {
+  position: absolute;
+  top: 0;
+  left: 0;
+  border: 0;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(148, 33, 33, 0.1);
+  background-image: linear-gradient(rgba(155, 148, 148, 0.2), rgba(0, 0, 0, 1));
+}
+
+.overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  border: 0;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(87, 86, 86, 0.2);
+  background-image: linear-gradient(to bottom, rgba(87, 86, 86, 0.2), rgba(0, 0, 0, 1));
+}
+
+.date {
+  display: none;
+}
+
+.tnw-img {
+  width: 100%;
+  height: 100%;
+}
+
+.caption {
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+  color: #fff;
+}
+
+.caption h3 a {
+  color: #fff;
+}
+
+.dropdown-content a:hover {
+  color: white;
+}
+
+.publication,
+span {
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+  font-weight: bold;
+  color: lightgrey;
+  gap: 10px;
+  margin: 4px 0;
+}
+
+.highlighicon {
+  width: 16px;
+  height: 16px;
+}
+
+@media screen and (min-width: 822px) {
+  div {
+    margin: 0;
+  }
+
+  .overlay-main {
+    position: absolute;
+    top: 0;
+    left: 0;
+    border: 0;
+    margin: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(148, 33, 33, 0.1);
+    background-image: linear-gradient(to bottom, rgba(87, 86, 86, 0.2), rgba(0, 0, 0, 1));
+  }
+
+  .overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    border: 0;
+    margin: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(87, 86, 86, 0.2);
+    background-image: linear-gradient(to bottom, rgba(87, 86, 86, 0.2), rgba(0, 0, 0, 1));
+  }
+
+  .highlights {
+    margin-top: 20px;
+    margin-bottom: 60px;
+    height: 300px;
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    gap: 5px;
+    top: 100px;
+  }
+
+  .main-feature {
+    margin: 0;
+    padding: 0;
+    height: 250px;
+  }
+
+  .main-feature h3 {
+    font-size: 30px;
+    margin: 0;
+    padding: 5px 0;
+  }
+
+  .sub-feature h3 {
+    font-size: 22px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .sub-feature {
+    display: flex;
+    flex-direction: column;
+    gap: 5px;
+    padding: 0;
+    height: 150px;
+  }
+
+  .sub-feature .tnw-card {
+    height: 122px;
+  }
+
+  .publication,
+  span {
+    font-size: 0.8rem;
+    display: flex;
+    align-items: center;
+    font-weight: bold;
+    gap: 5px;
+  }
+}
+
+@media screen and (min-width: 1024px) {
```

Comment:
[Optional]
- We have a repeating media query!! Thus `line 489` and `line 705`. Kindly fix this